receiver/otlpreceiver/factory.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlpreceiver
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+	"github.com/open-telemetry/opentelemetry-collector/consumer"
+	"github.com/open-telemetry/opentelemetry-collector/receiver"
+)
+
+const (
+	// The value of "type" key in configuration.
+	typeStr = "otlp"
+)
+
+// Factory is the Factory for receiver.
+type Factory struct {
+}
+
+// Type gets the type of the Receiver config created by this Factory.
+func (f *Factory) Type() string {
+	return typeStr
+}
+
+// CustomUnmarshaler returns nil because we don't need custom unmarshaling for this config.
+func (f *Factory) CustomUnmarshaler() receiver.CustomUnmarshaler {
+	return nil
+}
+
+// CreateDefaultConfig creates the default configuration for receiver.
+func (f *Factory) CreateDefaultConfig() configmodels.Receiver {
+	return &Config{
+		SecureReceiverSettings: receiver.SecureReceiverSettings{
+			ReceiverSettings: configmodels.ReceiverSettings{
+				TypeVal:  typeStr,
+				NameVal:  typeStr,
+				Endpoint: "localhost:55678",
```

Comment:
Let's use 55680 (OpenCensus+2) as the default port for OTLP.
We will need to also add this to OTLP spec.
 /cc @bogdandrutu

Reply:
Done.